app/Controllers/statsController.php

```diff
@@ -66,15 +66,15 @@ public function indexAction(): void {
 
 		$feedByCategory = [];
 		$feedByCategory_calculated = $statsDAO->calculateFeedByCategory();
-		for ($i = 0; $i < count($feedByCategory_calculated); $i++) {
+		for ($i = 0, $iMax = count($feedByCategory_calculated); $i < $iMax; $i++) {
```

Comment:
This does not look good. What do you try to do here?

Reply:
Ah, is it for an optimisation. Then I suggest either a count do to zero (reverse loop) if the order does not matter (needs to be checked), or a variable before.

Reply:
It does not look related at all to this PR about improving PHPDoc, though

Reply:
Ping ;-)